docker-compose.prod.yml

```
@@ -0,0 +1,158 @@
+
+  # Laravel Horizon
+  horizon:
+    image: ${ECR_REGISTRY}/${ECR_REPOSITORY}:${IMAGE_TAG}
+    container_name: laravel-horizon
+    restart: unless-stopped
+    environment:
+      - CONTAINER_ROLE=horizon
+      - APP_ENV=production
+      - APP_DEBUG=false
+      - APP_KEY=${APP_KEY}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)